### PR TITLE
Document new KinematicCollision2D class

### PR DESCRIPTION
**Pull request by cbscribe**
Preliminary docs for new KinematicCollision2D class. Since it was recently added, hopefully this will help as people are testing it.

Eventually needs a good code example as well.

### CODE REVIEW

File: doc/classes/KinematicBody2D.xml [hunk 1]
```diff
@@ -63,6 +63,7 @@
 			<argument index="0" name="rel_vec" type="Vector2">
 			</argument>
 			<description>
+				Moves the body along the given vector.  The body will stop if it collides.  Returns a [KinematicCollision2D], which contains information about the colliding body.
```
Comment (Noshyaar) [inline doc/classes/KinematicBody2D.xml]: Double spaces.

File: doc/classes/KinematicCollision2D.xml [hunk 2]
```diff
@@ -78,26 +81,37 @@
 	</methods>
 	<members>
 		<member name="collider" type="Object" setter="" getter="get_collider">
+			The colliding body.
 		</member>
 		<member name="collider_id" type="int" setter="" getter="get_collider_id">
+			The colliding body's unique [RID].
 		</member>
 		<member name="collider_metadata" type="Variant" setter="" getter="get_collider_metadata">
+			The colliding body's metadata.  See [Object].
```
Comment (Noshyaar) [inline doc/classes/KinematicCollision2D.xml]: Double spaces here too.

File: doc/classes/KinematicCollision2D.xml [hunk 3]
```diff
@@ -78,26 +81,37 @@
...
 		</member>
 		<member name="collider_shape" type="Object" setter="" getter="get_collider_shape">
+			The colliding body's shape.
 		</member>
 		<member name="collider_shape_index" type="int" setter="" getter="get_collider_shape_index">
+			The colliding shape's index.  See [CollisionObject2D].
```
Comment (Noshyaar) [inline doc/classes/KinematicCollision2D.xml]: Also here.

Comment (cbscribe) [reply doc/classes/KinematicCollision2D.xml]: Removed.

**Pull request merged**